encode_utils/connection.py

```diff
@@ -230,7 +230,7 @@ def auth(self):
         environment variables via the ``_get_api_keys_from_env()`` private instance
         method.
         """
-        if self._auth is None:
+        if not self._auth:
```

Comment:
I think it would be better to leave this line as is, then change https://github.com/StanfordBioinformatics/encode_utils/blob/4cde56ff9418177bf4297c4ec0b6695f0e4eb623/encode_utils/connection.py#L138 to initially set `self._auth = None`.